components/ai_chat/resources/page/state/ai_chat_context.tsx

```diff
@@ -17,20 +15,13 @@ interface Props {
   onSelectConversationUuid: (id: string | undefined) => unknown
 }
 
-export interface AIChatContext extends Props {
-  visibleConversations: mojom.Conversation[]
-  hasAcceptedAgreement: boolean
-  isPremiumStatusFetching: boolean
-  isPremiumUser: boolean
-  isPremiumUserDisconnected: boolean
-  canShowPremiumPrompt?: boolean
-  isMobile: boolean
-  isStandalone?: boolean
-  isHistoryEnabled: boolean
-  allActions: mojom.ActionGroup[]
+export type AIChatContext = Props & mojom.UIState & {
```

Comment:
Its a bit weird that this is `mojom.UIState` when the `UIState` interface isn't really part of the mojom API but a thing we're using to wrap some state